cypress/TestCases/FireboltCertification/Device.feature

```diff
@@ -67,3 +67,59 @@ Feature: Device
+
+    @Device @mfos @coreSDK @regression @sdk
+    Scenario Outline: Device.<Method> - Negative Scenario: <Scenario>
+        When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with '<validation_key>'
+
+        Examples:
+            | Scenario                                             | Method                 |  API_Key                                                                   | validation_key                                     |
+            | Invalid Audio codec type integer                     | audioFormatSupported   | get audioFormatSupported with invalid Audiocodec type integer              | invalid parameter error for audioFormatSupported   | 
+            | Invalid Audio codec type boolean                     | audioFormatSupported   | get audioFormatSupported with invalid Audiocodec type boolean              | invalid parameter error for audioFormatSupported   |
+            | Invalid audioFormatOptions atmos type interger       | audioFormatSupported   | get audioFormatSupported with audioFormatOptions atmos type interger       | invalid parameter error for audioFormatSupported   |
+            | Invalid audioFormatOptions atmos type string         | audioFormatSupported   | get audioFormatSupported with audioFormatOptions atmos type string         | invalid parameter error for audioFormatSupported   |
+            | Invalid Video codec type integer                     | videoFormatSupported   | get videoFormatSupported with invalid Videocodec type integer              | invalid parameter error for videoFormatSupported   |
+            | Invalid Video codec type boolean                     | videoFormatSupported   | get videoFormatSupported with invalid Videocodec type boolean              | invalid parameter error for videoFormatSupported   |
+            | Invalid VideoFormatOptions resolution type boolean   | videoFormatSupported   | get videoFormatSupported with VideoFormatOptions resolution type boolean   | invalid parameter error for videoFormatSupported   |
+            | Invalid VideoFormatOptions resolution type interger  | videoFormatSupported   | get videoFormatSupported with VideoFormatOptions resolution type integer   | invalid parameter error for videoFormatSupported   |
+
+    @Device @coreSDK @sdk @transport @notSupported
+    Scenario Outline: Device.<method> - Positive Scenario: <Scenario>
+        When '3rd party app' registers for the '<Event_Registration_Key>' event using the 'Firebolt' API
+        And '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
+        Then 'Firebolt' platform responds with '<Method_Validation_Key>'
+        When User triggers event with value as '<Event_Param>'
+        Then 'Firebolt' platform triggers event '<Event_Validation_Key>'
+
```

Comment:
For all TP test cases, we don't have all the different combinations currently. That has to be added. 
For eg, device.network has TC which returns the API response in all supported formats. 
Similarly, test cases have to be added for all possible combinations

Reply:
Added more combination testcases